code/game/gamemodes/cult/runes/convert.dm

```diff
@@ -70,20 +70,36 @@
-				//waiting_for_input ensures this is only shown once, so they basically auto-resist from here on out. They still need to find a way to get off the freaking rune if they don't want to burn to death, though.
-				to_chat(target, SPAN_CULT("Your blood pulses. Your head throbs. The world goes red. All at once you are aware of a horrible, horrible truth. The veil of reality has been ripped away and in the festering wound left behind, something sinister takes root."))
-				to_chat(target, SPAN_CULT("And you were able to force it out of your mind. Though the memory of that dark, horrible vision will surely haunt you for decades to come."))
-				target.visible_message(SPAN_WARNING("The markings below [target] lose their glow, this unworthy offering has been rejected!"))
+			if(!cult.can_become_antag(target.mind) || player_is_antag(target.mind))
+				if(jobban_isbanned(target.mind, "cultist"))
+					shard_player(target, A)
+				else
+					//waiting_for_input ensures this is only shown once, so they basically auto-resist from here on out. They still need to find a way to get off the freaking rune if they don't want to burn to death, though.
+					to_chat(target, SPAN_CULT("Your blood pulses. Your head throbs. The world goes red. All at once you are aware of a horrible, horrible truth. The veil of reality has been ripped away and in the festering wound left behind, something sinister takes root."))
+					to_chat(target, SPAN_CULT("And you were able to force it out of your mind. Though the memory of that dark, horrible vision will surely haunt you for decades to come."))
+					target.visible_message(SPAN_WARNING("The markings below [target] lose their glow, this unworthy offering has been rejected!"))
 			else
-				var/choice = alert(target,"Do you want to join the cult?", "Submit to Nar'Sie", "Resist", "Submit")
+				var/choice = alert(target,"Do you want to join the cult? (Choosing ghost will ghost you and spawn a shade)", "Submit to Nar'Sie", "Resist", "Submit", "Ghost")
 				waiting_for_input[target] = FALSE
-				if(choice == "Submit") //choosing 'Resist' does nothing of course.
-					cult.add_antagonist(target.mind)
-					converting -= target
-					target.hallucination = 0 //sudden clarity
-					target.setBrainLoss(0) // nar'sie heals you
-					sound_to(target, 'sound/effects/bloodcult.ogg')
+				switch(choice)
+					if("Submit")
+						cult.add_antagonist(target.mind)
+						converting -= target
+						target.hallucination = 0 //sudden clarity
+						target.setBrainLoss(0) // nar'sie heals you
+						sound_to(target, 'sound/effects/bloodcult.ogg')
+					if("Ghost")
+						shard_player(target, A)
+
 		sleep(100) //proc once every 10 seconds
 	LAZYCLEARLIST(converting)
 	return TRUE
+
+/datum/rune/convert/proc/shard_player(var/mob/living/target, atom/movable/A)
+	converting -= target
+	var/obj/item/device/soulstone/stone = new /obj/item/device/soulstone(get_turf(A))
+	target.death()
```

Comment:
this should ash them

Reply:
the death's just so the soulstone can work, so i'm gonna add .dust() after everything's done and sorted out